content/vault/v1.21.x (rc)/content/docs/license/product-usage-reporting.mdx

```diff
@@ -109,145 +109,152 @@ HashiCorp collects the following product usage metrics as part of the `metrics`
+| `vault.adaptive_overload_protection_enabled`                        | Whether or not Adaptive Overload Protection is enabled for the cluster.                                    |
+| `vault.policies.egp.count`                                          | The total number of Enterprise Governance Policies (EGP) across all namespaces in Vault.                   |
+| `vault.policies.rgp.count`                                          | The total number of Replication Governance Policies (RGP) across all namespaces in Vault.                  |
+| `vault.policies.acl.count`                                          | The total number of Access Control List (ACL) policies across all namespaces in Vault.                     |
+| `vault.policies.acl.control_group.count`                            | The total number of control groups in this cluster.                                                        |
+| `vault.autopilot_upgrade_enabled`                                   | Whether or not Autopilot Upgrade is enabled for the cluster.                                               |
+| `vault.audit.device.file.count`                                     | The total number of audit devices of type file configured for this cluster.                                |
+| `vault.audit.device.syslog.count`                                   | The total total number of audit devices of type syslog configured for this cluster.                        |
+| `vault.audit.device.socket.udp.count`                               | The total total number of audit devices of type UDP socket configured for this cluster.                    |
+| `vault.audit.device.socket.tcp.count`                               | The total total number of audit devices of type TCP socket configured for this cluster.                    |
+| `vault.audit.device.socket.unix.count`                              | The total total number of audit devices of type UNIX socket configured for this cluster.                   |
+| `vault.audit.exclusion.stanza.count`                                | The total number of audit exclusion stanzas in the HCL configuration for this cluster.                     |
+| `vault.loadedsnapshots.manual.count`                                | The total number of manual (local) snapshots loaded for recovery.                                          |
+| `vault.loadedsnapshots.cloud.google-gcs.count`                      | The total number of Google GCS snapshots loaded for recovery.                                              |
+| `vault.loadedsnapshots.cloud.aws-s3.count`                          | The total number of AWS S3 snapshots loaded for recovery.                                                  |
+| `vault.loadedsnapshots.cloud.azure-blob.count`                      | The total number of Azure Blob snapshots loaded for recovery.                                              |
```

Comment:
I think this file might have some issues. The line numbers don't match up with the file on v1.20.x and in spot checking, I found a few discrepancies (I didn't make it all the way through the table).